src/gpu/intel/jit/v2/conv/kernel_desc.hpp

```diff
@@ -506,6 +508,7 @@ struct trivial_key_validator_t<jit::v2::conv::kernel_desc_t> {
                 && (t.thread_group_tile == tmp.thread_group_tile)
                 && (t.loop_desc == tmp.loop_desc)
                 && (t.prefetch == tmp.prefetch) && (t.align == tmp.align)
+                && (t.use_stream_k == tmp.use_stream_k)
```

Comment:
I don't see where this gets serialized? Am I just missing it?

Reply:
It's picked up automatically from the parse interface, through this line:

https://github.com/oneapi-src/oneDNN/blob/b6ede45d1ef0250167e817fc879c0927f42ece41/src/gpu/intel/jit/v2/conv/kernel_desc.cpp#L440